fixtures/uking/mini-spider-light.json

```
@@ -0,0 +1,317 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Mini Spider Light",
+  "shortName": "UK-MiniSpider",
+  "categories": ["Effect"],
+  "meta": {
+    "authors": ["Anonymus", "CGBassPlayer"],
+    "createDate": "2023-03-21",
+    "lastModifyDate": "2023-04-24"
```

Please update both dates to today. The create date should be the date where the fixture is released on OFL (i.e. where the PR is merged).

```suggestion
    "createDate": "2023-04-24",
    "lastModifyDate": "2023-04-24"
```